docs/src/examples/collections/Table/Variations/TableExampleSortable.js

```diff
@@ -1,62 +1,51 @@
 import _ from 'lodash'
-import React, { Component } from 'react'
-import { Table } from 'semantic-ui-react'
+import React, { useState } from "react";
+import { Header, Table } from "semantic-ui-react";
 
 const tableData = [
-  { name: 'John', age: 15, gender: 'Male' },
-  { name: 'Amber', age: 40, gender: 'Female' },
-  { name: 'Leslie', age: 25, gender: 'Other' },
-  { name: 'Ben', age: 70, gender: 'Male' },
-]
+  { name: "John", age: 15, gender: "Male" },
```

Comment:
It looks that you have a different formatting there. Please run `yarn prettier:fix` & `yarn lint:fix` it should fix your issues and pass build.